packages/flutter_app/.gitignore

```diff
@@ -77,7 +77,6 @@ unlinked_spec.ds
 **/android/.settings
 **/android/.project
 **/android/service-account.json
-**/android/app/src/main/res/mipmap*/
```

Comment:
`res.srcDirs = ['src/main/res', "src/${dartDefines.flavor}/res"]`, so this one is no longer needed✨